src/Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis.Tests/FunctionalRedisCacheTests.cs

```diff
@@ -0,0 +1,533 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using AutoFixture.AutoFakeItEasy;
+using AutoFixture.Xunit2;
+using FluentAssertions;
+using Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis.Tests._Extensions;
+using Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis.Tests._Models;
+using Functional.Primitives.FluentAssertions;
+using Xunit;
+
+namespace Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis.Tests
+{
+	// ReSharper disable once InconsistentNaming
+	public class FunctionalRedisCacheTests
+	{
+		// -- IMPORTANT --
```

Comment:
Maybe?  I figure if the tests fail (because Redis isn't running locally), the developer will jump to the tests to see why they fail and will then see the huge comment block that tells them what needs to happen.  If Redis is not running, the tests will fail with the following exception:
```
ServiceStack.Redis.RedisException : localhost:6379
---- System.Net.Internals.SocketExceptionFactory+ExtendedSocketException : No connection could be made because the target machine actively refused it 127.0.0.1:6379
```

So far, the markdown documentation is just cookbook-type stuff.